.eslintrc

```diff
@@ -3,21 +3,32 @@
   "parserOptions": {
     "sourceType": "module"
   },
-  "plugins": ["jest-formatting", "jest"],
+  "plugins": ["jest-formatting", "jest", "@typescript-eslint/eslint-plugin"],
   "env": {
     "jest/globals": true
   },
   "extends": [
     "airbnb-base",
+    "plugin:@typescript-eslint/recommended",
     "plugin:prettier/recommended",
-    "plugin:jest-formatting/recommended"
+    "plugin:jest-formatting/recommended",
+    "plugin:import/typescript"
   ],
   "overrides": [
     {
       "files": ["**.*"],
       "rules": {
-        "camelcase": "off"
+        "camelcase": "off",
+        "no-plusplus": "off",
```

Comment:
What's the intention behind this override? Is this coming from one of the rule set extensions and adding this override prevents a breaking change?

Reply:
I think you nailed it. Looked through Dan's commits and this one seems to have come in from one of the rule sets. We could track down our use of plusplus and then re-enable this rule in a patch release maybe.

Reply:
Yah, there is a minusminus used, but I figured it was out of scope for now. I was thinking it's worth it to do a 'fix all linting overrides' PR at some point.